Taskfile.yaml

```diff
@@ -56,3 +56,24 @@ tasks:
       - '{{ .BUILD_ROOT }}/bin/go/gofumpt -l -w {{.DIRECTORY}}'
       - '{{ .BUILD_ROOT }}/bin/go/gci write -s default -s standard -s "prefix(github.com/redpanda-data/common-go)" {{.DIRECTORY}}'
       - if [[ $CI == "true" ]]; then git --no-pager diff --exit-code; fi
+
+  test:
+    desc: Run all tests
+    deps:
+      - 'fmt'
+      - 'lint'
+    cmds:
+      - for: { var: MODULES }
+        task: test:dir
+        vars:
+          DIRECTORY: '{{.ITEM}}'
+  
+  test:dir:
```

Comment:
It's already a transitive dependency of the parent `task test`, but I'd add `install:go` as a dependency explicitly here